openqaoa/optimizers/training_vqa.py

```diff
@@ -283,15 +283,15 @@ def results_dictionary(self,
                 #. "solution"
                     #. "bitstring"
                     #. "degeneracy"
-                #. "number of evals"
-                #. "jac evals"
-                #. "qfim evals"
+                #. "number_of_evals"
+                #. "jac_evals"
+                #. "qfim_evals"
                 #. "parameter log"
```

Comment:
Why is there a space here?

Reply:
I think these shouldn't be there, they are not in the results

Reply:
I eliminated that part because in that function we are not returning anything